docs/user_guides/moons_reporter.md

```diff
@@ -0,0 +1,129 @@
+![SeAT](https://i.imgur.com/aPPOxSK.png)
+
+# Moons Reporter
+
+Since SeAT 4.0, a new tool is appear to assist you in your moon management task by leveraging your intelligence desk.
+Like your old google/excel sheet, it will gather all your intel related to moon probing.
+It provides a centralized and convenient way to store reports and make search in them.
```

Comment:
```suggestion
Introduced in SeAT 4.0, a new tool is available to assist you in your moon management task by leveraging your intelligence desk.
Like your old google/excel sheet, it will gather all your intel related to moon probing.
It provides a centralized and convenient way to store reports and search through them.
```